Make.config

```
@@ -140,8 +140,8 @@ endif
 ##
 ## Note that the prerelease identifier should be as short as possible, because otherwise
 ## the resulting package name can become too long for MSIs.
-NUGET_HARDCODED_PRERELEASE_IDENTIFIER=net8-rtm
-NUGET_HARDCODED_PRERELEASE_BRANCH=net8.0
+NUGET_HARDCODED_PRERELEASE_IDENTIFIER=xcode16
+NUGET_HARDCODED_PRERELEASE_BRANCH=xcode16
```

@rolfbjarne Shouldn't this be reverted to net8.0?

 ```suggestion
NUGET_HARDCODED_PRERELEASE_IDENTIFIER=net8-rtm
NUGET_HARDCODED_PRERELEASE_BRANCH=net8.0
```

---

It doesn't really matter, because `NUGET_HARDCODED_PRERELEASE_IDENTIFIER` has no effect unless `NUGET_HARDCODED_PRERELEASE_BRANCH` matches the current branch (which it won't anymore, because we're on `main`).

In any case I updated it as you suggested.